homeassistant/components/amcrest/sensor.py

```diff
@@ -89,32 +75,34 @@ def update(self):
         """Get the latest data and updates the state."""
         if not self.available:
             return
-        _LOGGER.debug("Updating %s sensor", self._name)
+        _LOGGER.debug("Updating %s sensor", self.entity_description.key)
 
         try:
-            if self._sensor_type == SENSOR_PTZ_PRESET:
-                self._state = self._api.ptz_presets_count
+            if self.entity_description.key == SENSOR_PTZ_PRESET:
+                self._attr_is_on = self._api.ptz_presets_count
```

Comment:
`_attr_is_on` is not part of the Sensor entity.

Reply:
My bad !!! I reacted on the review comment from ludeeus but that was another file.